ui/src/app/views/workspace/component.spec.ts

```diff
@@ -110,6 +110,7 @@ describe('WorkspaceComponent', () => {
         { provide: BugReportService, useValue: new BugReportServiceStub() },
         { provide: ClusterService, useValue: new ClusterServiceStub() },
         { provide: CohortsService, useValue: new CohortsServiceStub() },
+        { provide: ConceptSetsService },
```

Comment:
Should this be using the stub?

Reply:
This is not needed as          { provide: ConceptSetsService, useValue: new ConceptSetsServiceStub() },
 is already below in the list